### PR TITLE
Prepare release notes for 3.1.1

### PR DESCRIPTION
**Pull request by hardys**
Prepare release notes for the planned 3.1.1 release

### CODE REVIEW

File: asciidoc/day2/migration.adoc [hunk 1]
```diff
@@ -525,7 +525,7 @@ _Use-case:_
 
 _Steps:_
 
-. Locate the desired versions for `Rancher` and `Metal^3^` from the <<release-notes, release notes>>. For `Edge 3.1.0`, these versions would be `2.9.1` for _Rancher_ and `0.8.1` for _Metal^3^_.
+. Locate the desired versions for `Rancher` and `Metal^3^` from the <<release-notes, release notes>>. For `Edge 3.1.0`, these versions would be `2.9.3` for _Rancher_ and `0.8.3` for _Metal^3^_.
```
Comment (fdegir) [inline asciidoc/day2/migration.adoc:528]:
```suggestion
. Locate the desired versions for `Rancher` and `Metal^3^` from the <<release-notes, release notes>>. For `Edge 3.1.1`, these versions would be `2.9.3` for _Rancher_ and `0.8.3` for _Metal^3^_.
```

**Review by fdegir — COMMENTED** [review]
Small issue with the release version, looks good otherwise.

File: asciidoc/edge-book/version-matrix.adoc [hunk 2]
```diff
@@ -17,11 +17,11 @@ endif::[]
 |======
 | Name | Version | Chart Version
 | SLE Micro | 6 | N/A
-| Rancher Prime | 2.9.1 | 2.9.1
+| Rancher Prime | 2.9.3 | 2.9.3
 | Fleet | 0.10.1 | 104.0.1+up0.10.1
```
Comment (ipetrov117) [inline asciidoc/edge-book/version-matrix.adoc:21]: Fleet version for Rancher `2.9.3` is `0.10.4`, see their [build.yaml](https://github.com/rancher/rancher/blob/v2.9.3/build.yaml#L5C15-L5C31) for more detail.

File: asciidoc/day2/migration.adoc [hunk 3]
```diff
@@ -196,7 +196,7 @@ For a full list of the options for the `drain` command, see link:https://kuberne
 +
```
Comment (ipetrov117) [inline asciidoc/day2/migration.adoc:196]: @hardys, It was my impression that we agreed that migration would be done from `3.0.z -> 3.1.0`,  where `3.0.z` must be the latest z-ed stream release. And that after that all upgrades would happen one version at a time using the `upgrade-controller`.

If my understanding is correct, I suggest we bring back the `3.1.0` versions in the `migration.adoc` documentation.

File: asciidoc/day2/migration.adoc (hunk 1 above)
Comment (hardys) [reply asciidoc/day2/migration.adoc:528]: Based on discussion with @ipetrov117 we won't update the migration docs to constrain the upgrade path from 3.0, so will close this thread

File: asciidoc/edge-book/releasenotes.adoc [hunk 4]
```diff
@@ -28,6 +28,110 @@ However, repeated entries are provided as a courtesy only. Therefore, if you are
...
+Availability Date: 15th November 2024
+
+Summary: SUSE Edge 3.1.1 is the first release z-stream in the SUSE Edge 3.1 release stream.
+
+== Bug & Security Fixes
+
+* The Rancher version is updated to `2.9.3`: https://github.com/rancher/rancher/releases/tag/v2.9.3[Release Notes]
+* The RKE2 version is updated to `1.30.5`: https://docs.rke2.io/release-notes/v1.30.X#release-v1305rke2r1[Release Notes]
+* The K3s version is updated to `1.30.5`:  https://docs.k3s.io/release-notes/v1.30.X#release-v1305k3s1[Release Notes]
+* The Metal^3^ chart fixes an issue where with the handling of the `predictableNicNames` parameter: https://github.com/suse-edge/charts/pull/160[SUSE Edge issue #160]
```
Comment (atanasdinov) [inline asciidoc/edge-book/releasenotes.adoc:42]:
```suggestion
* The Metal^3^ chart fixes an issue with the handling of the `predictableNicNames` parameter: https://github.com/suse-edge/charts/pull/160[SUSE Edge issue #160]
```

Comment (atanasdinov) [inline asciidoc/edge-book/releasenotes.adoc:42]: This one and the following are not technically "issues" but PRs. Is this something that we want to adjust or is the current wording fine?

File: asciidoc/edge-book/releasenotes.adoc [hunk 5]
```diff
@@ -28,6 +28,110 @@ However, repeated entries are provided as a courtesy only. Therefore, if you are
 
 NOTE: SUSE Edge z-stream releases are tightly integrated and thoroughly tested as a versioned stack. Upgrade of any individual components to a different versions to those listed above is likely to result in system downtime. While it's possible to run Edge clusters in untested configurations, it is not recommended, and it may take longer to provide resolution through the support channels.
 
+= Release 3.1.1
+
+Availability Date: 15th November 2024
+
+Summary: SUSE Edge 3.1.1 is the first release z-stream in the SUSE Edge 3.1 release stream.
+
+== Bug & Security Fixes
```
Comment (atanasdinov) [inline asciidoc/edge-book/releasenotes.adoc:37]: We have to add the Turtles update here, as well as add separate category for "new features" and mention the NeuVector 5.4 update there.

File: asciidoc/edge-book/releasenotes.adoc (hunk 4 above)
Comment (hardys) [reply asciidoc/edge-book/releasenotes.adoc]: I just copied the same convention used in previous releases - we didn't have public issues for all of these, so pointing to the PR for more context seemed like the most reasonable alternative

File: asciidoc/edge-book/releasenotes.adoc (hunk 5 above)
Comment (hardys) [reply asciidoc/edge-book/releasenotes.adoc:41]: Thanks, done!

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**